SPECS/dracut/20overlayfs/overlayfs-mount.sh

```diff
@@ -51,6 +51,11 @@ parse_kernel_cmdline_args() {
 
     # Retrieve the OverlayFS parameters.
     [ -z "${overlayfs}" ] && overlayfs=$(getarg rd.overlayfs=)
+
+    # Retrieve the optional OverlayFS mount options from the kernel cmdline.
```

Comment:
Lanze, is it possible to add a readme for this module? define it's interface etc